addons/mail/static/src/components/thread_view_topbar/thread_view_topbar.js

```diff
@@ -24,6 +24,24 @@ export class ThreadViewTopbar extends Component {
     // Public
     //--------------------------------------------------------------------------
 
+    /**
+     * Determine whether description should display on top bar.
+     *
+     * @returns {boolean}
+     */
+    get hasThreadViewTopbarDescription() {
```

Comment:
turn this into a field on `ThreadViewTopbar` model, rather than getter on the component.